### ISSUE TITLE
Dropdown on the homepage dataviz is missing the focus indicator

### ISSUE THREAD

**Issue by lizzycederberg** [usnationalarchives/nara-bia-photos#257]
Dropdown on the homepage dataviz is missing the focus indicator. (related to #254 and #256)
